### PR TITLE
freecad: new, 0.20.2

### PR DESCRIPTION
**Pull request by Icenowy**
Topic Description
-----------------

Introduce FreeCAD and its dependencies， introducing downgrading an existing package to a broadly used version.

Package(s) Affected
-------------------

- `opencascade`: downgrade
- `kicad`: rebuild for opencascade downgrading
- `med coin`: new packages

Security Update?
----------------

No

Build Order
-----------

`opencascade kicad med coin freecad`

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: extra-creativity/freecad/spec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VER=0.20.2
+SRCS="git::commit=tags/${VER}::https://github.com/FreeCAD/FreeCAD"
+CHKSUMS="SKIP"
```
Comment (MingcongBai) [inline extra-creativity/freecad/spec:3]: Missing CHKUPDATE.

File: extra-libs/med/spec [hunk 2]
```diff
@@ -0,0 +1,5 @@
+VER=4.1.1
+SRCS="tbl::https://files.salome-platform.org/Salome/other/med-$VER.tar.gz"
+CHKSUMS="sha256::dc2b5d54ebf0666e3ff2e974041d2ab0da906061323537023ab165d573389dd0"
+# No CHKUPDATE here because its upstream now has no information about
+# this package, and it's not yet available on Anitya.
```
Comment (MingcongBai) [inline extra-libs/med/spec:5]: Use "folder" or "custom" mode in Anitya.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-libs/med/spec (hunk 2 above)
Comment (Icenowy) [reply extra-libs/med/spec:5]: The problem is lack of update information.

Comment (Icenowy) [reply extra-libs/med/spec:5]: The folder itself is not browsable, and the website has now no info about this library, only their suites.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**